<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICHCDetectionFrame</class>
 <widget class="QFrame" name="ICHCDetectionFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="ICComboBox" name="detectFixture2ComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Reversed Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Positive Phase</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ICComboBox" name="detectFixture3ComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Reversed Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Positive Phase</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Ejection Link Lock</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Detect fixture 2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ICComboBox" name="detectIMMComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Not Use</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="ICComboBox" name="ejectionLinkLockBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Not Use</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="ICComboBox" name="detectMidMoldComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Use</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not Use</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Detect Mid Mold</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="ICComboBox" name="detectPressureComboBox">
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Not Use</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Detect fixture 4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Detect fixture 3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Detect Pressure</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>IMM Emergency</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ICComboBox" name="detectFixture4ComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Reversed Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Positive Phase</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ICComboBox" name="detectFixture1ComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Reversed Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Positive Phase</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Detect fixture 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Detect Security Door</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="ICComboBox" name="detectSecurityComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Not Use</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="ICComboBox" name="detectSucker2ComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Not Use</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Auto Run</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Close Mold</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="ICComboBox" name="detectSucker1ComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Not Use</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Hor Standby</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="ICComboBox" name="standbyPositionBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>No Limit Lock Mold</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Limit Lock Mold</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>U/D Position</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="ICComboBox" name="originPositionBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Vertical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Horizontal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Limit</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Detect Origin</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="ICComboBox" name="detectOriginBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Don't need Mold Opened Signal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Need Mold Opened Signal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Detect Postion</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="ICComboBox" name="detectPositionBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>48</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Vertical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Horizontal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Don't Limit</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICComboBox</class>
   <extends>QComboBox</extends>
   <header>iccombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
